dev/com.ibm.ws.security.jaspic_fat/fat/src/com/ibm/ws/security/jaspic11/fat/audit/JASPICallbackBasicAuthAuditTest.java

```diff
@@ -65,6 +71,12 @@
 @RunWith(FATRunner.class)
 @Mode(TestMode.FULL)
 public class JASPICallbackBasicAuthAuditTest extends JASPITestBase {
+    @ClassRule
+    public static RepeatTests r = EERepeatActions.repeat("com.ibm.ws.security.jaspic11.fat.callback.audit",
+    EERepeatActions.EE10, // EE10
+    EERepeatActions.EE9, // EE9
+    EERepeatActions.EE8).with(new EmptyAction().fullFATOnly())
+    .andWith(new FeatureReplacementAction(AuditCommonTest.AUDIT_1_0_FEATURES, AuditCommonTest.AUDIT_2_0_FEATURES));
```

Comment:
```suggestion
    /**
     * Need the first repeat to make sure that audit-2.0 from a previous repeat gets put back to audit-1.0
     */
    @ClassRule
    public static RepeatTests auditRepeat = RepeatTests.with(new FeatureReplacementAction("audit-2.0", "audit-1.0"))
                    .andWith(new FeatureReplacementAction("audit-1.0", "audit-2.0"));
```
Don't need the EE repeat since it is already done in FATSuite.